src/components/BadgeSelect.tsx

```diff
@@ -0,0 +1,98 @@
+import has = require('lodash/has');
+import includes = require('lodash/includes');
+import * as React from 'react';
+import { BadgeSelectProps } from 'rendition';
+import styled from 'styled-components';
+import Badge from './Badge';
+import DropDownButton from './DropDownButton';
+import Text from './Text';
+
+interface BadgeSelectState {
+	selected: string | null | undefined;
+}
+
+const ButtonWrapper = styled.button`
+	border: 0;
+	background: none;
+	display: block;
+	margin-top: 5px;
+	margin-bottom: 5px;
+	text-align: left;
+	min-width: 182px;
+	padding-left: 10px;
+	padding-right: 10px;
+	padding-top: 3px;
+	padding-bottom: 3px;
```

Comment:
I think that we also need to define a cursor in here.

Reply:
:+1: